README.md

```diff
@@ -150,4 +150,14 @@ Now that you have a basic sense of how to read these error messages, as well as
 
 When you're done, run `learn` to confirm all tests are passing.
 
+#### What's the Difference between `gets`, `gets.chomp`, and `gets.strip`?
+
+When getting user input, you may want to remove whitespace. The `gets.chomp` method removes whitespace AFTER the user input. The `gets.strip` method removes whitespace BEFORE and AFTER the user input. The `gets` method by itself does not remove any whitespace or newlines from the user input. Your tests will pass using any of these methods, however, be aware that they perform different actions and are not always interchangeable for every use-case.
```

Comment:
It's not true that `chomp` rm's whitespace (` `, for example, it does _not_ remove). 

>  chomp also removes carriage return characters (that is it will remove \n, \r, and \r\n). If $/ is an empty string, it will remove all trailing newlines from the string.

```ruby
2.3.3 :005 > x = "Hi     "
 => "Hi     "
2.3.3 :006 > x.length
 => 7
2.3.3 :007 > x.chomp.length
 => 7
```

Reply:
"...AFTER the input (including the carriage return characters)."

Seems useful to introduce the idea of carriage return characters and their symbols.

Maybe some code examples showing IRB output might be good here?

Maybe these 3 methods should be broken into sub-headings?

Maybe summarize their effects in a table  "  this is a test \n" => how do chomp, strip, etc. vary it?

Reply:
Based on this paragraph, it seems like `gets` should come first ; explain that it's "dirty"; explain it an be cleaned by chomp or strip.